rp2040-hal/examples/blinky.rs

```diff
@@ -0,0 +1,29 @@
+//! Blinks the LED on a Pico board
+//!
+//! This will blink an LED attached to GP25, which is the pin the Pico uses for the on-board LED.
+#![no_std]
+#![no_main]
+
+use cortex_m_rt::entry;
+use embedded_hal::digital::v2::OutputPin;
+use panic_halt as _;
+use rp2040_hal::prelude::*;
+
+#[link_section = ".boot2"]
+#[used]
+pub static BOOT2: [u8; 256] = rp2040_boot2::BOOT_LOADER;
+
+#[entry]
+fn main() -> ! {
+    let mut pac = rp2040_pac::Peripherals::take().unwrap();
+
+    let pins = pac.IO_BANK0.split(pac.PADS_BANK0, pac.SIO, &mut pac.RESETS);
+    let mut led_pin = pins.gpio25.into_output();
+
+    loop {
+        led_pin.set_low().unwrap();
+        // TODO: I dare not use delays until we've got clocks running
```

Comment:
You could use cortex_m::asm::delay for this purpose
It does mean blink speed is arbitrary, but that's better than blinking so fast that no-one can perceive it